src/index.js

```diff
@@ -400,6 +400,19 @@ function createConfig(options, entry, format, writeMeta) {
 	const absMain = resolve(options.cwd, getMain({ options, entry, format }));
 	const outputDir = dirname(absMain);
 	const outputEntryFileName = basename(absMain);
+	let tsconfigPath;
+	let tsconfigOptions = {};
+	let ts;
+	if (useTypescript) {
+		tsconfigPath = resolve(
+			options.tsconfig || resolve(options.cwd, 'tsconfig.json'),
+		);
+		ts = require(resolveFrom.silent(options.cwd, 'typescript') || 'typescript');
+		const tsconfigJSON = ts.readConfigFile(tsconfigPath, ts.sys.readFile)
```

Comment:
You'll need to check to ensure the tsconfig actually exists. I think a user can get by using Microbundle without one, thanks to the tsconfig defaults

Reply:
This will also be relevant down in the `esModule` change too